lib/compiler/webpack-compiler.ts

```diff
@@ -85,12 +86,12 @@ export class WebpackCompiler {
       console.log(statsOutput);
     };
 
-    if (watchMode) {
+    if (watchMode || webpackConfiguration.watch) {
       compiler.hooks.watchRun.tapAsync('Rebuild info', (params, callback) => {
         console.log(`\n${INFO_PREFIX} Webpack is building your sources...\n`);
         callback();
       });
-      compiler.watch({}, afterCallback);
+      compiler.watch(webpackConfiguration.watchOptions!, afterCallback);
```

Comment:
```suggestion
      compiler.watch(webpackConfiguration.watchOptions! || {}, afterCallback);
```

Perhaps we should fallback here to the default value